test/bigtest/network/scenarios/stabbedLoanPolicies.js

```diff
@@ -0,0 +1,68 @@
+export default (server) => {
+  server.get('/loan-policy-storage/loan-policies', {
```

Comment:
I don't think this belongs to scenario. Shouldn't this be defined in `config.js`? Also the factory should be used for generating them. Then we can use the scenario to create a couple of these records in a similar fashion to: https://github.com/folio-org/ui-checkin/blob/master/test/bigtest/network/scenarios/default.js#L4-L5

Reply:
Done.